kalkulator.py

```diff
@@ -4,9 +4,14 @@ def dodaj(a,b):
     wynik=a+b
     return wynik
     
+def odejmij(a,b):
+    return a - b
```

Comment:
Moim zdaniem tutaj lepiej rozbić to na dwie linijki wprowadzając zmienną "wynik".